src/Http/Http/src/WriteOnlyPipeStream.cs

```diff
@@ -81,9 +81,17 @@ public override void Flush()
         }
 
         /// <inheritdoc />
-        public override async Task FlushAsync(CancellationToken cancellationToken)
+        public override Task FlushAsync(CancellationToken cancellationToken)
         {
-            await InnerPipeWriter.FlushAsync(cancellationToken);
+            var task = InnerPipeWriter.FlushAsync(cancellationToken);
```

Comment:
Can this use the `.GetAsTask()` extension instead?

Reply:
Place in a shared file in https://github.com/aspnet/AspNetCore/tree/master/src/Shared and reuse here, servers, and SignalR

Reply:
Yeah I think it deserves to be shared now. I will once I’m not OOF.